### PR TITLE
refactor(core): separate session routes and UTs

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Separate session routes and UTs.

## Linear Issue Reference

LOG-2061
LOG-2062

## Testing

UTs.

### CODE REVIEW

**Review by demonzoo — APPROVED** [review]

File: packages/core/src/routes/session/passwordless.test.ts [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+    });
+  });
+
+  describe('POST /session/sign-in/passwordless/email/send-passcode', () => {
+    beforeAll(() => {
+      interactionDetails.mockResolvedValue({
+        jti: 'jti',
+      });
+    });
+    it('it should call sendPasscode', async () => {
```
Comment (IceHe) [inline packages/core/src/routes/session/passwordless.test.ts:121]: nit: `"it"` is unnecessary in the description

File: packages/core/src/routes/session/passwordless.test.ts [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+    ],
+  });
+
+  describe('POST /session/sign-in/passwordless/sms/send-passcode', () => {
+    beforeAll(() => {
+      interactionDetails.mockResolvedValueOnce({
+        jti: 'jti',
+      });
+    });
+    it('it should call sendPasscode', async () => {
```
Comment (IceHe) [inline packages/core/src/routes/session/passwordless.test.ts:72]: nit: `"it"` is unnecessary in the description

File: packages/core/src/routes/session/passwordless.test.ts [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+  });
+
+  describe('POST /session/register/passwordless/sms/send-passcode', () => {
+    beforeAll(() => {
+      interactionDetails.mockResolvedValueOnce({
+        jti: 'jti',
+      });
+    });
+
+    it('it should call sendPasscode', async () => {
```
Comment (IceHe) [inline packages/core/src/routes/session/passwordless.test.ts:171]: ditto

File: packages/core/src/routes/session/passwordless.test.ts [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+      );
+      expect(interactionResult).toHaveBeenCalledWith(
+        expect.anything(),
+        expect.anything(),
+        expect.objectContaining({ login: { accountId: 'user1' } }),
+        expect.anything()
+      );
+    });
+
+    it('throw error if phone not valid (characters other than digits)', async () => {
```
Comment (IceHe) [inline packages/core/src/routes/session/passwordless.test.ts:219]: nit: not valid → is invalid

File: packages/core/src/routes/session/passwordless.test.ts [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+  });
+
+  describe('POST /session/register/passwordless/email/send-passcode', () => {
+    beforeAll(() => {
+      interactionDetails.mockResolvedValueOnce({
+        jti: 'jti',
+      });
+    });
+
+    it('it should call sendPasscode', async () => {
```
Comment (IceHe) [inline packages/core/src/routes/session/passwordless.test.ts:255]: ditto

**Review by IceHe — APPROVED** [review]
LGTM

**Pull request merged**